### ISSUE TITLE
An issue with the latest version naming debs in latest release

### ISSUE THREAD

**Issue by sevenpastzeero** [SarahH12099/Windows-Repo-Tool#9]
Hi,

In v2.0.0, if you open a repo and download a few debs, their names are correct. Then if you open a second repo after the first one, downloaded debs names are either from the first repo, or from the second repo but with incorrect tweak name.
I hope I made sense.

**Comment by SarahH12099** [comment]
Yeah you made sense, I am currently without a computer so I will ask Lillie (@LillieWeeb) to try to fix this tomorrow for you. Sorry about that.

**Comment by meadestephen74-crypto** [comment]
> Yeah you made sense, I am currently without a computer so I will ask Lillie (@LillieWeeb) to try to fix this tomorrow for you. Sorry about that.
